src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Create the build
+weight: 20
+---
+
+# Create a build package
+
+Once preliminary tasks have been completed, the project is ready to be built.
+
+## 1. Communicate
+
+Starting this step means that the Development phase of this release is over. Before you go further, make sure to tick the "Development" box in the Release Tracker GitHub issue (there is one per version, see the [1.7.6.6 example][release-tracker-issue]).
+
+## 2. Create a local branch for your build
+
+The following tasks will require you to perform changes and submit them as a Pull Request.
+
+* **Clone the project** on your computer using Git (only if you don't already have a local copy of the repository).
+
+    ```shell
+    git clone git@github.com:PrestaShop/PrestaShop.git
+    ```
+
+* **Make sure that you switch to the appropriate branch** regarding the version you'll be building (e.g. you must be on branch 1.7.7.x if you're building a 1.7.7 release).
+
+* **Make sure your branch is up-to-date with upstream.** Especially if you already had a local clone of the repository.
+
+* **Create a local branch for you work.** Keep it! You will need to go back to it later. 
+
+## 3. Merge any not-yet published security fixes
+
+{{% notice note %}}
+**About security advisories.**
+
+To avoid disclosing security issues, security Pull Request are merged _after_ the build has been done and validated. In order to include them in your build, you need to retrieve them manually and merge them in your local branch.
+{{% /notice %}}
+
+If this release includes security PRs:
+
+- Add the different private temporary repositories for each Security Advisory as remotes of your local git repository.
+- Merge each one of those PRs into your working local branch.
+
+
+## 4. Update the Changelog & Contributors list
+
+### Use the changelog tool to extract the data
+
+{{% notice warning %}}
+**This step requires special rights.**
+
+Ask a maintainer from the PrestaShop Company with access to the Changelog Tool to perform this step.
+{{% /notice %}}
+
+After this step, you should obtain two files:
+
+- the Changelog file – including a list of all the merged Pull Requests. Make sure to keep this file, you'll need it later.
+- the New Contributors file – a list of all the people who contributed code for this version for their first time.
+
+### Update the files in the project
+
+- Add the contents of the changelog at the top end of PrestaShop's [Changelog file][changelog-file].
+- Update PrestaShop's [Contributors file][contributors-file] by adding the new contributors.
+- Commit your changed files with following message: `// Changelog [version]`
+- You may push this branch to your own fork, if you wish to.
```

Comment:
Maybe on a **private** fork if it contains security fixes?

Reply:
```suggestion
- You may push this branch to your own (private) fork, if you wish to.
```